clusterloader2/pkg/errors/fatal_error.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"errors"
+	"fmt"
+)
+
+var ErrFatal = errors.New("fatal error")
```

Comment:
As we already have `isErrsCritical` let's call it `ErrCritical` as well.
Or rename `isErrsCritical` to `isErrsFatal`

Reply:
Good idea, I replaced `fatal` with `critical`